ui/components/app/edit-gas-fee-button/edit-gas-fee-button.js

```diff
@@ -71,7 +71,7 @@ export default function EditGasFeeButton({ userAcknowledgedGasMissing }) {
           </span>
         )}
         <span className="edit-gas-fee-button__label">{t(title)}</span>
-        <i className="fas fa-chevron-right asset-list-item__chevron-right" />
```

Comment:
You will have to import the `Icon` component and `ICON_NAMES` object and check class name incase more styles are necesary

```
import { Icon, ICON_NAMES } from '../../component-library';
```